crates/hyperswitch_connectors/src/connectors/bambora/transformers.rs

```diff
@@ -173,13 +177,37 @@ impl TryFrom<BamboraRouterData<&types::PaymentsAuthorizeRouterData>> for Bambora
                     complete: item.router_data.request.is_auto_capture()?,
                 };
 
+                let province = item
+                    .router_data
+                    .get_optional_billing_state()
+                    .and_then(|state| {
+                        if state.clone().expose().len() > 2 {
+                            None
+                        } else {
+                            Some(state)
+                        }
+                    });
```

Comment:
Can this be reused from the utils function to get 2 digit state code?

Reply:
yes, this is reusable. I moved it to connector utils